# Local position in NED frame

uint64 timestamp	# timestamp of the setpoint
float32 x		# in meters NED
float32 y		# in meters NED
float32 z		# in meters NED
float32 yaw		# in radians NED -PI..+PI
